OMEdit/OMEditLIB/Modeling/Model.cpp

```diff
@@ -777,15 +779,14 @@ namespace ModelInstance
       if (!subModifiers.isEmpty()) {
         value.append("(" % subModifiers.join(", ") % ")");
       }
-      if (mValue.isEmpty()) {
-        if (value.compare(mName) != 0) {
-          return value;
-        } else {
-          return "";
-        }
-      } else {
-        return value.append(mName.isEmpty() ? mValue : " = " % mValue);
+      if (!mValue.isEmpty()) {
+        value.append(mName.isEmpty() ? mValue : " = " % mValue);
       }
+      if (!mComment.isEmpty()) {
```

Comment:
Maybe this is not needed. We just want the comment for display purposes. We don't want to add it to the modification.

Reply:
I added it there since choices are stored as Modifier:s. But I guess it will break if we decide to dump comments for normal modifiers too, so maybe it would be better to handle comments separately as you say.

Reply:
Storing them as Modifiers is fine. But I will dump them separately just for display purpose.
I will test few use cases and then update it accordingly.